src/com/rcv/ResultsWriter.java

```diff
@@ -137,7 +137,7 @@ void generatePrecinctSummarySpreadsheets(
       // the precinct output filename
       String precinctFileString = getPrecinctFileString(precinct, filenames);
       // filename for output
-      String outputFileName = String.format("%s_%s.xlsx", this.timestamp, precinctFileString);
+      String outputFileName = String.format("%s_%s_precinct_summary.xlsx", this.timestampString, precinctFileString);
```

Comment:
For sorting purposes, maybe naming this `"%s_summary_precinct_%s.xlsx"` would be better?

Also, I can't easily verify in GitHub here, but can we verify that `precinctFileString` doesn't already have the word "precinct" in it?